tools/nl/eval/README.md

```diff
@@ -9,3 +9,41 @@ python3 fetch_explore_queries.py
 
 This produces a csv file that can be imported to Google Sheet for further manual
 curation to produce golden data for eval purpose.
+
+## Prepare Model and Embedding Endpoints
+
+To evaluate a model and embeddings generated from the model, it needs to be
+deployed to Vertex AI. Concrete steps can be found in
+[model server](../../../model_server/README.md).
+
+Once a model is deployed, generate embeddings and index them in Vertex AI vector
+search. Concreate steps can be found in [build
+embeddings](../embeddings/README.md).
+
+Once model and embeddings endpoints are avaiable, record the endpoints id in
+`vertex_ai_endpoints.yaml` file.
+
+## Eval Unit
+
+One evaluation unit locates in a sub folder that contains a `golden.json` file
+that is a mapping between golden query and ranked stat vars to be matched from
+the vector store. Note there is logic to get to ranked stat vars from ranked
+embedding vectors. Right now it's simply based on the stat vars for a
```

Comment:
I'm not fully sure what "logic to get to ranked stat vars from ranked embedding vectors" means?

Reply:
Updated here to reflect what we do in nl server logic. Basically looked at the highest ranked description for a stat var.